### DISCUSSION TITLE
Ability to create project groups to facilitate navigation between projects.

### DISCUSSION THREAD

**Discussion by Malkverbena** [godotengine/godot-proposals#4229] [Editor]
We often have large projects that have several other smaller projects.
See the example of a multiplayer.
We have one project for the server and one for the client and several other projects for possible tools used in the general scope.
My proposal is based on being able to create project groups.
Instead of the project list displaying all projects at once, it can display groups of projects. This way, the navigation through the projects will become simpler.

**Comment by Calinou** [comment]
There's already a proposal tracking this: https://github.com/godotengine/godot-proposals/issues/1133
